.github/workflows/cli-build.yml

```diff
@@ -0,0 +1,44 @@
+name: CLI
```

Comment:
Should we give this a more specific name, e.g `CLI Build`?

Reply:
I see. I guess we can keep it as is then.

Reply:
Added specific naming for now